src/client.ts

```diff
@@ -3,11 +3,29 @@ import { metricsDefinitions } from './metrics'
 import { SnapshotsFetcherComponents } from './types'
 import { contentServerMetricLabels, fetchJson, saveContentFileToDisk as saveContentFile } from './utils'
 
-export async function getGlobalSnapshot(components: SnapshotsFetcherComponents, server: string, retries: number) {
-  const url = new URL(`${server}/snapshot`).toString()
+export async function getGlobalSnapshots(components: SnapshotsFetcherComponents, server: string, retries: number):
+  Promise<{
+    hash: string,
+    lastIncludedDeploymentTimestamp: number
+  }[]> {
+  try {
+    const newUrl = new URL(`${server}/snapshots`).toString()
```

Comment:
can we name this variable something like incrementalSnapshots or snapshotsRanges ?

Reply:
yes